File: clases/semilleros/competitiva/otros/Beautiful_Matrix.py
#encontrar 1 y ver distancia al centro,
mat=[]
for i in range(5):
	ent=input().split()
	mat.append(ent)
	if "1" in ent:
		for ii in range(5):
				if "1"== ent[ii]:
					print(abs(i-2)+abs(ii-2))
					
					
#print(mat)

#print(getdis(x,maxv=5)+getdis(y,maxv=5)-1)

File: clases/semilleros/competitiva/otros/Beautiful_Matrix.cpp
#include <bits/stdc++.h>
using namespace std;
int main(){
	int in,dis;
	int n=5;
	int mat[n][n];
	for(int i=0;i<n;i++){
		for(int j=0;j<n;j++){
			cin>>in;
			if(in==1){
				//dis=abs(i-3)+abs(j-3);
				cout<<abs(i-2)+abs(j-2)<<endl;
				break;
                //cout<< abs(i-3)+abs(j-3)<<endl;
			}				
		}
	}
}